x/lend/keeper/keeper.go

```diff
@@ -110,6 +110,22 @@ func (k Keeper) CheckSupplyCap(ctx sdk.Context, assetID, poolID uint64, amt sdk.
 	return false, nil
 }
 
+func (k Keeper) ReturnLendID(ctx sdk.Context, lenderAddr string, AssetID, PoolID uint64) uint64 {
```

Comment:
can it be `GetLendIDForAssetIDPoolID` instead of `ReturnLendID`? Let's be more specific about the method names with the tasks they perform.